### ISSUE TITLE
Leadership page modal links use invalid role

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#15538]
# Description

There's a small remaining a11y issue on the current `/about/leadership` page. We should not be using `role="button"` on a landmark like `<article>` content - instead it should just be a regular `<div>`. It seems OK that all descendants are declared as presentational when the role is applied, as only the person's name is read aloud which seems OK.

We should also add `aria-controls` and `aria-expanded`, to associate control with the modal element.

## axe-core found 1 violation

| #   | Description | Axe rule ID | WCAG | Impact | Count |
| --- | --- | --- | --- | --- | --- |
| [1](#1) | ARIA role should be appropriate for the element | aria-allowed-role | Best practice | minor | 1   |

### Failed

1. ARIA role should be appropriate for the element

[Learn more](https://dequeuniversity.com/rules/axe/4.10/aria-allowed-role?application=playwright)

aria-allowed-role

Best practice

Ensure role attribute has an appropriate value for the element

minor

Issue Tags: cat.aria best-practice

| #   | Issue Description | To solve this violation, you need to... |
| --- | --- | --- |
| 1   | **Element location**<br><br>```<br>#laura-chambers<br>```<br><br>**Element source**<br><br>```<br><article id="laura-chambers" class="vcard has-bio" itemscope="" itemtype="http://schema.org/Person" role="button" tabindex="0"><br>``` | Fix any of the following:<br><br>*   ARIA role button is not allowed for given element |
